docs/en/ufs/HDFS.md

```diff
@@ -192,9 +200,18 @@ ALLUXIO_JAVA_OPTS+=" -Djava.security.krb5.realm=<YOUR_KERBEROS_REALM> -Djava.sec
 ### Mount HDFS with Specific Versions
 
 There are multiple ways for a user to mount an HDFS cluster with a specified version as an under storage into Alluxio namespace.
+Before mounting HDFS with a specific version, make sure you have built a client with that specific version of HDFS.
```

Comment:
after 2.0, we support 1.2, 2.2 and 2.7 by default, for these versions you don't need to build hdfs.
for other hdfs versions, we need to build it as you indicated here. I think we can perhaps use some different hadoop version to illustrate this, e.g. some cdf versions? also show how to mount this cdf hdfs ufs connector

Reply:
if they are building from source, I don't think 2.7 gets built by default. I was using 2.7 and had to go through this step to get it working.  Maybe just tell them to check that the correct jar exists and build if it does not exist.